test/public/flps/overview.test.js

```diff
@@ -89,10 +98,14 @@ module.exports = () => {
 
     it('Should display the correct items counter at the bottom of the page', async () => {
         await goToPage(page, 'flp-overview');
-        await waitForTimeout(100);
 
+        await page.waitForSelector('#firstRowIndex');
         expect(await page.$eval('#firstRowIndex', (element) => parseInt(element.innerText, 10))).to.equal(1);
+
+        await page.waitForSelector('#lastRowIndex');
         expect(await page.$eval('#lastRowIndex', (element) => parseInt(element.innerText, 10))).to.equal(10);
+
+        await page.waitForSelector('#totalRowsCount');
         expect(await page.$eval('#totalRowsCount', (element) => parseInt(element.innerText, 10))).to.equal(105);
     });
```

Comment:
Same, you can directly use expectInnerText

Reply:
Done